bootstrap/components/sinks/kafka-sink-topology-component.json

```diff
@@ -72,10 +72,11 @@
       {
```

Comment:
We might want to keep this `writerSchemaVersion` below `topic` field to make a better user experience.